web-app/js/portal/lang/en.js

```diff
@@ -228,6 +226,7 @@ OpenLayers.Lang.en = OpenLayers.Util.extend(OpenLayers.Lang.en, {
     depthLabel: 'Depth:',
     elevationLabel: 'Elevation:',
 
+    showHideTooltip: '{0} from the map',
     removeDataCollectionTooltip: 'Remove this data collection from the Subset and Download pages, the collection can be re-added from the search',
```

Comment:
Sorry should have mentioned this before - you happy with "Hide from map"  and "Show from map" as the hints?